crates/pallets/starknet/src/runtime_api.rs

```diff
@@ -46,7 +46,7 @@ sp_api::decl_runtime_apis! {
         /// Returns the chain id.
         fn chain_id() -> Felt252Wrapper;
         /// Returns fee estimate
```

Comment:
Add in the doc, which `u64` is the `overall_fee` and which is the `gas_consumed`

Reply:
But even better would be creating a dedicated struct so the API users are less prone to commit mistakes